DevelopmentProposals/template.md

```diff
@@ -0,0 +1,63 @@
+## Project: TITLE
+***OpenSim Design Document***
+
+### Overview
+
+Just a few sentences that describes the purpose of this project. For example:
+Create a graphical tool to plot simulation data.
+
+### Requirements
+
+List the set of requirements that this project must fulfill. Include all constraints this proposal is working within.
+If the list gets too long, consider splitting the project into multiple small projects.
+
+For example:
+
+1. GUI should plot values over time, where values can be joint angles, poses of objects, forces on objects, diagnostic signals, and values from topics.
```

Comment:
I think the template should talk about a project that is more likely for `opensim-core`; this repo doesn't have a GUI.